translator/main.py

```diff
@@ -366,38 +396,40 @@ def create_dashboard_cm(
             args_.jsonnet_dashboards_selector,
             target_dashboards_selector,
         )
-    data = {}
+
     for filename, json_data in jsons:
         if cms.items and filename in dashboards_skip:
             log.info(f"Update of {filename} dashboard skipped")
             continue
 
+        data = {}
         data[filename] = json.dumps(json_data)
-
-    configmap = client.V1ConfigMap(
-        data=data,
-        metadata=client.V1ObjectMeta(
-            name=name,
-            namespace=namespace,
-            labels=labels,
-            annotations=user_annotations,
-        ),
-    )
-    # Create/Update ConfigMap
-    if cms.items:
-        try:
-            v1.replace_namespaced_config_map(name, namespace, configmap)
-        except ApiException as e:
-            log.error(f"Error when replacing config map {name}, error {e}")
-            raise RetryException
-        log.info(f"Dashboards resource {name} updated")
-    else:
-        try:
-            v1.create_namespaced_config_map(namespace, configmap)
-        except ApiException as e:
-            log.error(f"Error when creating config map {name}, error {e}")
-            raise RetryException
-        log.info(f"Dashboards resource {name} created")
+        name = "-".join([args_.grafana_dashboards_cm_name, filename[:-5]])
```

Comment:
What is this `filename[:-5]`? I suppose it removes `.json`, but it is not obvious

Reply:
Yes, exactly it removes `.json`

Reply:
Can you leave a comment there?

Reply:
Yes